cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -222,6 +222,7 @@ arrow::Status VeloxShuffleWriter::init() {
 
   ARROW_ASSIGN_OR_RAISE(
       partitioner_, Partitioner::make(options_.partitioning, numPartitions_, options_.startPartitionId));
+  DLOG(INFO) << "Create partitioning type: " << std::to_string(options_.partitioning);
```

Comment:
this looks like a debug log?

Reply:
Yes. The `DLOG` ensures that it only get printed with debug mode. We probably need this log for debugging, because sometimes the shuffle operator can get omitted on Spark UI, such as a single partitioning after limit operator.